# this has joint poses in degrees and dt in seconds

float32 dt
int16 min_joint_angle
int16 max_joint_angle
int16 joint1pos
int16 joint2pos